src/pint/models/model_builder.py

```diff
@@ -134,6 +137,16 @@ def __call__(
             converted to TDB upon read. If "raw", an unconverted malformed TCB
             TimingModel object will be returned.
 
+        allow_T2 : True, False, or str, optional
+            Whether to convert a T2 binary model to an appropriate underlying
+            binary model. Default is False, and will throw an error upon
+            encountering the T2 binary model. If True, the binary model will be
+            converted to the most appropriate PINT-compatible binary model.
```

Comment:
What does an str value for this parameter do?

Reply:
Changed the docstring